### PR TITLE
Show more information on shippable parcels screen

### PR DESCRIPTION
**Pull request by edsinkevic**
scrn-2023-05-30-17-18-42

### CODE REVIEW

**Comment by SimonasKalpokas1** [comment]
I don't think sender's and receiver's names and the code of the parcel are necessary to display here, as they are irrelevant to the courier (in case he needs to deliver to a terminal).

File: Colorblind/Domain/DTOs/ShippableParcelInTerminal.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+using Domain.Entities;
+
+namespace Domain.DTOs;
+
+public record ShippableParcelInTerminal(
+    Parcel Parcel,
```
Comment (SimonasKalpokas1) [inline Colorblind/Domain/DTOs/ShippableParcelInTerminal.cs:6]: We probably don't care about this, but this kinda leaks sensitive data, like phone numbers

Comment (edsinkevic) [reply Colorblind/Domain/DTOs/ShippableParcelInTerminal.cs:6]: I agree, forget about this PR. I really wanted this page to show more information, but I didn't realize that it's irrelevant.

**Review by SimonasKalpokas1 — APPROVED** [review]
Nice

File: Colorblind/Persistence/Implementation/ParcelRepository.cs [hunk 2]
```diff
@@ -42,8 +42,7 @@ public ParcelRepository(IDocumentSession documentSession)
 
         return parcels
             .Select(x =>
-                new ParcelInTerminalDTO(x.Id, x.Code, x.Version,
-                    destinations[x.ReceiverDeliveryInfo.TerminalId].Address))
+                new ShippableParcelInTerminal(x, destinations[x.ReceiverDeliveryInfo.TerminalId]))
```
Comment (SimonasKalpokas1) [inline Colorblind/Persistence/Implementation/ParcelRepository.cs:45]: Maybe mapping to the response could happen here, but maybe the current way is better not sure

**Pull request merged**